code/game/objects/items/blackout.dm

```diff
@@ -0,0 +1,84 @@
+// Blackout tool, used to trigger massive electricity outttage on ship or station, including connected levels.
+// It may have additional shots to use, but currently balanced to one shot.
+
+/datum/uplink_item/item/tools/blackout
+	name = "High Pulse Electricity Outage Tool"
+	item_cost = 24
+	path = /obj/item/blackout
+	desc = "A device wich can create power virus in terminal, spread it in power network and temporally creating blackout."
+
+/obj/item/blackout
+	name = "high pulse electricity outage tool"
+	desc = "A unknown device, probably only experienced electrics know what this can do."
+	icon = 'icons/obj/items/blackout.dmi'
+	icon_state = "device_blackout-off"
+
+	var/severity = 2
+	var/shots = 1
+	var/lastUse = 0
+	var/cooldown = (20 MINUTES)
+
+/obj/item/blackout/afterattack(obj/target, mob/user, proximity)
+	if(!proximity)
+		return
+
+	if(!istype(target))
+		return
+
+	add_fingerprint(user)
+
+	if(istype(target, /obj/machinery/power/terminal))
+		var/obj/machinery/power/terminal/terminal = target
+
+		if(!terminal.powernet)
+			to_chat(user, SPAN_WARNING("This power station isn't connected to power net."))
+			return
+
+		if(check_to_use())
+			to_chat(user, SPAN_WARNING("Device does not respond. Perhaps you need to try later."))
+			return
+
+		if(!shots)
+			to_chat(user, SPAN_WARNING("Device does not respond."))
+			return
+
+		hacktheenergy(terminal, user)
+
+/obj/item/blackout/proc/hacktheenergy(obj/machinery/power/terminal/terminal_in, mob/user)
+	if(!istype(terminal_in) || !user) return
+
+	src.audible_message("<font color=Maroon><b>HackTheEnergy.exe Assistant</b></font> says, \
```

Comment:
The hyphens are a bit weird in a spoken message.
Also `terminal` not therminal.